### DISCUSSION TITLE
[Script request] Wallabag

### DISCUSSION THREAD

**Discussion by clartek** [community-scripts/ProxmoxVE#437] [Request script]
### Application Name

Wallabag

### Website

https://wallabag.org/

### Description

Read it later app

### Due Diligence

- [X] I have searched existing [scripts](https://community-scripts.github.io/Proxmox/scripts) and found no duplicates.
- [X] I have searched existing [discussions](https://github.com/community-scripts/ProxmoxVE/discussions?discussions_q=) and found no duplicate requests.
- [X] This is not a game-related request.

**Comment by MickLesk** [comment]
https://github.com/community-scripts/ProxmoxVED/issues/1196

The wallabag script is ready for testing:
```bash -c "$(curl -fsSL https://github.com/community-scripts/ProxmoxVED/raw/main/ct/wallabag.sh)"```

Database credentials are stored in ~/wallabag.creds

Note: This is not in the official repo yet—it's just a dev version! After merging into ProxmoxVE, it will need to be recreated.
